README.md

```diff
@@ -23,14 +23,12 @@ J. Blow.
 
 Required:
 
-- OCaml 4.06.1
+- OCaml 4.08 or later
```

Comment:
FYI: I could not get master to build now with 4.08.1. 4.13.1 worked though.

Reply:
For me 4.08 does not show anymore in the available switches, and 4.10 works.
Probably some feature introduced later is used, which is OK.
No plan to support specific old versions. Will just update the README.